py-polars/polars/eager/series.py

```diff
@@ -1929,6 +2010,31 @@ def rolling_max(
         min_periods
             The number of values in the window that should be non-null before computing a result.
             If None, it will be set equal to window size.
+
+        Examples
+        --------
+        >>> s = pl.Series("a", [100, 200, 300, 400, 500])
+        >>> s.rolling_max(window_size=2)
+        shape: (5,)
+        Series: '' [i64]
+        [
+                null
+                null
+                300
+                400
+                500
+        ]
+        >>> s.rolling_max(window_size=3, min_periods=2))
```

Comment:
Maybe show an example that differs from output from the first one?

Reply:
I am going to leave second examples out. I don't find them very clear either. Will come back to it when I have played around with it and found a better example